packages/components/psammead-episode-list/src/index.stories.jsx

```diff
@@ -0,0 +1,23 @@
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
```

Comment:
Storybook and psammead-storybook-helpers should be in peer dependencies in package.json